setup.cfg

```diff
@@ -8,7 +8,7 @@ ignore =
 
 [metadata]
 requires-dist =
-    clint
+    tqdm
```

Comment:
This needs to match `setup.py`. When people install from a wheel, we want the requirements to match those that they see when installing from an sdist.